--- automake-1.12.5/automake.in.orig	2012-11-22 17:36:51.560603115 +0100
+++ automake-1.12.5/automake.in	2012-11-22 18:18:52.190550575 +0100
@@ -29,7 +29,7 @@
 
 BEGIN
 {
-  @Automake::perl_libdirs = ('@datadir@/@PACKAGE@-@APIVERSION@')
+  @Automake::perl_libdirs = ('@datadir@/@PACKAGE@')
     unless @Automake::perl_libdirs;
   unshift @INC, @Automake::perl_libdirs;
 
--- automake-1.12.5/aclocal.in.orig	2012-11-22 17:36:51.597269780 +0100
+++ automake-1.12.5/aclocal.in	2012-11-22 18:19:11.647216837 +0100
@@ -27,7 +27,7 @@
 
 BEGIN
 {
-  @Aclocal::perl_libdirs = ('@datadir@/@PACKAGE@-@APIVERSION@')
+  @Aclocal::perl_libdirs = ('@datadir@/@PACKAGE@')
     unless @Aclocal::perl_libdirs;
   unshift @INC, @Aclocal::perl_libdirs;
 }
--- automake-1.12.1/lib/Automake/Config.in.orig	2012-06-01 15:47:09.000000000 +0200
+++ automake-1.12.1/lib/Automake/Config.in	2012-06-03 07:49:50.581740616 +0200
@@ -32,7 +32,7 @@
 our $PACKAGE_BUGREPORT = '@PACKAGE_BUGREPORT@';
 our $VERSION = '@VERSION@';
 our $RELEASE_YEAR = '@RELEASE_YEAR@';
-our $libdir = '@datadir@/@PACKAGE@-@APIVERSION@';
+our $libdir = '@datadir@/@PACKAGE@';
 our $perl_threads = @PERL_THREADS@;
 
 1;
